test/address_sanitizer_setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+OLD_DIR=`pwd`
+
+cd "$SCRIPT_DIR/.."
+
+if [ ! -d otp ]
+then
+    git clone git@github.com:erlang/otp.git
+
+    cd otp
+
+    export ERL_TOP=`pwd`
+    export PATH=$ERL_TOP/bin:$PATH
+    
+    echo "Changed ERL_TOP and PATH"
+
+
+    git checkout OTP-24.2.1
+
+    LAST_COMMIT_SHA=$(git rev-parse HEAD)
+
+    if [ ${LAST_COMMIT_SHA} != 7bf7f01683acf9b8f09bd8c7331854a9abc17f7d ]
```

Comment:
Suggestion: maybe we could directly checkout this commit instead of failing?

Reply:
Yes, but then it might not be as clear which Erlang release we are checking out (this could be fixed with a comment however).